### DISCUSSION TITLE
DIA data quantification - precursor level - UPS1 proteins

### DISCUSSION THREAD

**Discussion by mlocardpaulet** [Proteobench/ProteoBench#211] [Potential new module to discuss]
The idea is to do the same module as the module `DDA data quantification - precursor ions` with the data from https://doi.org/10.1021/acs.jproteome.1c00490
Then we can, like with the DDA module, make the same module at peptidoform level.

**Comment by mlocardpaulet** [comment]
Here is a copy of previous notes that we took:

PROVIDED DATA:
1. Raw files of DIA runs (triplicates): 1-10fmol (we don't want to run all the files of the paper). These contain a background of lysate of e. coli and spike-ins (known quantities of 48 purified human proteins (UPS1))
2. Fasta file:
Fasta file with protein sequences:
UniprotKB/SwissProt of e. coli
The 48 UPS1 proteins
Retention time standards used in the runs
Contaminants 

The figure would be the same as in the DDA module.

**Reply by wolski** [reply]
UPS dataset is available here 
https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD026600
MassIVE MSV000087597

**Comment by mlocardpaulet** [comment]
Comments/concerns from previous meetins:
spectral library will have a big impact on the workflow output. Shall we enforce the use of specific libraries? 
Shall we use only one predicted library? The same for all workflows? We could provide a library in different formats for users to try out. 

-> For the moment we only tried out "direct" searches with in-silico library generation from the fasta file.

**Comment by wolski** [comment]
We could also use the triple proteome data because we reuse most of the code from DDA modules and allow for "cross comparison" between DDA and DIA.
https://www.nature.com/articles/s41597-022-01216-6

**Reply by mlocardpaulet** [reply]
This is a really good idea. I'll start a discussion dedicated to such a module: discussion #257.
